static/sass/_pattern_strip.scss

```diff
@@ -0,0 +1,201 @@
+@mixin multipass-p-strip {
+  @include multipass-p-strip-suru;
+}
+
+@mixin multipass-p-strip-suru {
+  .p-strip {
+    overflow-x: hidden;
+  }
+
+  $suru-accent: #3fc8f2;
+
+  .p-strip--suru {
```

Comment:
We have these in Vanilla now. I think you just need to update the suru colors [like this](https://github.com/canonical-web-and-design/jp.ubuntu.com/blob/78e639a9529f1ae5a9ebe19e20791c076ba56ad1/static/sass/_settings_colors.scss#L8)